src/controller/bankrupt/bankrupt.controller.ts

```diff
@@ -33,7 +34,7 @@ export const postSearchPage = async (req: Request, res: Response, next: NextFunc
     // Not found officers has to be rendered anyway with an empty list 
     if(!results.error || results.status === 404){
       const { itemsPerPage = 0, startIndex = 0, totalResults = 0, items = [] } = results.data || {};
-      return res.render('bankrupt', { itemsPerPage, startIndex, totalResults, items: formattingOfficersInfo(items), searched: true });
+      return res.render('bankrupt', { itemsPerPage, startIndex, totalResults, items: formattingOfficersInfo(items), searched: true , userEmail});
     } else {
       return res.status(results.status).render('error-pages/500');
     } 
```

Comment:
Retrieve the `userEmail = userSession.getLoggedInUserEmail(req.session); ` when is needed, inside the if condition, and remove spaces please (line 9 and 39). Thanks :)